### ISSUE TITLE
Neglecting to check a box in advanced mws results in ignored gui inputs

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#2240]
If you attempt an ''Advanced Multi-Window Save'' from File->Set Save Options, you will see two check boxes at the top; one for ''Do Advanced Multi-Window Save'' and one for ''Stereo''. Neither is checked by default. But, withou one being checked, you can wind up entering gui controls for the individual windows and only after trying several times do you discover all your inputs are being ignored.

I recommend just checking the ''Advanced Multi-Window Save'' by default to avoid these issues.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Mark Miller
Original creation: 07/01/2015 10:41 pm
Original update: 09/10/2015 07:56 pm
Ticket number: 2330
